src/sage/modular/modform/element.py

```diff
@@ -233,9 +265,111 @@ def __call__(self, x, prec=None):
 
             sage: f(0)
             0
-        """
+
+        Evaluate numerically::
```

Comment:
```suggestion
        EXAMPLES:

        Evaluate numerically::
```

Reply:
@tscrim Wait, are you sure this is a good idea? Because there's already a `EXAMPLES::` a few lines before.

Reply:
Ah, sorry, I misread it. No, we shouldn't have it (plus I had it overly indented...). Please revert and disregard.